internal/service/ch/ch_test.go

```diff
@@ -35,29 +37,26 @@ func TestCHService(t *testing.T) {
 func (c *CHServiceTestSuite) SetupSuite() {
 	ctx := context.Background()
 	var err error
-	c.container, err = clickhouseinfra.CreateClickHouseContainer(ctx, "", "")
+	c.container, err = container.CreateClickHouseContainer(ctx, chconfig.Settings{})
 	c.Require().NoError(err, "Failed to create clickhouse container")
 
-	db, err := c.container.GetClickhouseAsDB(ctx)
+	db, err := c.container.GetClickhouseAsDB()
 	c.Require().NoError(err, "Failed to get clickhouse connection")
 
-	err = migrations.RunGoose(ctx, []string{"up", "-v"}, db)
-	c.Require().NoError(err, "Failed to run migrations")
-
-	host, err := c.container.Host(ctx)
-	c.Require().NoError(err, "Failed to get clickhouse host")
+	cfg := c.container.Config()
 
-	port, err := c.container.MappedPort(ctx, clickhouseinfra.SecureNativePort)
-	c.Require().NoError(err, "Failed to get clickhouse port")
+	err = migrate.RunGoose(ctx, []string{"up", "-v"}, migrations.RegisterFuncs(), db)
```

Comment:
We don't need to change this line. `migrations.RunGoose` is still a function in model-garage